includes/class-amp-reader-themes.php

```diff
@@ -0,0 +1,512 @@
+			'version'        => '3.1',
+			'preview_url'    => 'https://wp-themes.com/twentytwelve',
+			'author'         =>
+			[
+				'user_nicename' => 'wordpressdotorg',
+				'profile'       => 'https://profiles.wordpress.org/wordpressdotorg',
+				'avatar'        => 'https://secure.gravatar.com/avatar/61ee2579b8905e62b4b4045bdc92c11a?s=96&d=monsterid&r=g',
+				'display_name'  => 'WordPress.org',
+			],
+			'screenshot_url' => '//ts.w.org/wp-content/themes/twentytwelve/screenshot.png?ver=3.1',
+			'rating'         => 92,
+			'num_ratings'    => '155',
+			'homepage'       => 'https://wordpress.org/themes/twentytwelve/',
+			'description'    => 'The 2012 theme for WordPress is a fully responsive theme that looks great on any device. Features include a front page template with its own widgets, an optional display font, styling for post formats on both index and single views, and an optional no-sidebar page template. Make it yours with a custom menu, header image, and background.',
+			'requires'       => '3.5',
+			'requires_php'   => '5.2.4',
+		],
+		[
+			'name'           => 'Twenty Eleven',
+			'slug'           => 'twentyeleven',
+			'version'        => '3.4',
+			'preview_url'    => 'https://wp-themes.com/twentyeleven',
+			'author'         =>
+			[
+				'user_nicename' => 'wordpressdotorg',
+				'profile'       => 'https://profiles.wordpress.org/wordpressdotorg',
+				'avatar'        => 'https://secure.gravatar.com/avatar/61ee2579b8905e62b4b4045bdc92c11a?s=96&d=monsterid&r=g',
+				'display_name'  => 'WordPress.org',
+			],
+			'screenshot_url' => '//ts.w.org/wp-content/themes/twentyeleven/screenshot.png?ver=3.4',
+			'rating'         => 94,
+			'num_ratings'    => '45',
+			'homepage'       => 'https://wordpress.org/themes/twentyeleven/',
+			'description'    => 'The 2011 theme for WordPress is sophisticated, lightweight, and adaptable. Make it yours with a custom menu, header image, and background -- then go further with available theme options for light or dark color scheme, custom link colors, and three layout choices. Twenty Eleven comes equipped with a Showcase page template that transforms your front page into a showcase to show off your best content, widget support galore (sidebar, three footer areas, and a Showcase page widget area), and a custom "Ephemera" widget to display your Aside, Link, Quote, or Status posts. Included are styles for print and for the admin editor, support for featured images (as custom header images on posts and pages and as large images on featured "sticky" posts), and special styles for six different post formats.',
+			'requires'       => false,
+			'requires_php'   => '5.2.4',
+		],
+	];
+
+	/**
+	 * Retrieves all AMP plugin options specified in the endpoint schema.
+	 *
+	 * @return array Formatted theme data.
+	 */
+	public function get_themes() {
+		if ( null !== $this->themes ) {
+			return $this->themes;
+		}
+
+		$themes   = $this->get_default_supported_reader_themes();
```

Comment:
Just saw there is a flag to get the data from WordPress.org themes API:

```suggestion
		$themes   = $this->get_default_supported_reader_themes( true );
```

Reply:
In a subsequent PR.

Reply:
I was actually using `true` here during most of development, including during local PHPUnit testing, and never had an issue with it. Are there circumstances where you think we could handle it this way and remove the hardcoded theme data?

Reply:
We should avoid the hard-coded data because then we get translated strings. The `themes_api()` function by default makes its request with a `locale` argument that is set to `get_locale()`. So we _should_ remove the hardcoded theme data.

Reply:
This is added reason why we should just pass through the original theme descriptions, and not try to craft our own short versions to fit in the UI. We need to have translated descriptions where possible.